plugins/available/kuzzle-plugin-cluster/lib/node.js

```diff
@@ -24,6 +24,7 @@ const Bluebird = require('bluebird');
 const debug = require('debug')('kuzzle:cluster');
```

Comment:
Feel so good to be able to modify the cluster directly in Kuzzle :grin:

Reply:
You can say that again :+1: